{{cookiecutter.github_repository}}/requirements/common.txt

```diff
@@ -11,7 +10,7 @@ uWSGI==2.0.15
 # -------------------------------------
 argon2-cffi==18.1.0
 django-environ==0.4.4
-django-sites==0.9
+-e git+https://github.com/niwinz/django-sites.git@9402fb9dccaac9421dc9368002bcedbf30e1a36a#egg=django-sites
```

Comment:
Is this stable enough?

Reply:
We have a new release for this today. 0.10 and i've updated to use the pypi release.